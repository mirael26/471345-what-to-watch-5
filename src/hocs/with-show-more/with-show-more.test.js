import React from "react";
import renderer from "react-test-renderer";

import withShowMore from "./with-show-more";

const MockComponent = () => {
  return (
    <div>
    </div>
  );
};

const MockComponentWrapped = withShowMore(MockComponent);

it(`Should withShowMore render correctly`, () => {
  const noop = () => {};
  const tree = renderer
    .create(
        <MockComponentWrapped
          filmsCount={6}
          onShowMoreButton={noop}
        />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
