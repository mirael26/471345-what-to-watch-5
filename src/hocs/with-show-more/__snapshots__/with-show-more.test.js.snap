// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should withShowMore render correctly 1`] = `<div />`;
